grouper/graph.py

```diff
@@ -215,14 +215,16 @@ def user_indexify(data):
     @staticmethod
     def _get_permission_metadata(session):
         '''
-        Returns a dict of groupname: { list of permissions }.
+        Returns a dict of groupname: { list of permissions }. Note
+        that disabled permissions are not included.
```

Comment:
i will leave it as is because currently there is very little reason to care about disabled permissions, i.e., only interested in those disabled permissions when in some special cases, e.g., debug/troubleshoot/one-off audit, etc.